src/App.js

```diff
@@ -15,6 +15,7 @@ function App() {
   const [currentTheme, setCurrentTheme] = useState();
   const [station, setStation] = useState();
   const [stationData, setStationData] = useState();
+  const [volume, setVolume] = useState(50);
```

Comment:
Instead of defaulting to 50 can we first get the volume and set it.

Reply:
for now we can keep it like this and when we create context for handling the player we can do it